ui/js/reactrevisions.jsx

```diff
@@ -18,7 +18,9 @@ const RevisionItem = (props) => {
     let email, name, userTokens, escapedComment, escapedCommentHTML, initialsHTML, tags;
 
     userTokens = props.revision.author.split(/[<>]+/);
-    name = userTokens[0].trim();
+    name = userTokens[0].trim().replace(/\w\S*/g,
+                                        function (txt) { return txt.charAt(0).toUpperCase() +
```

Comment:
You should be able to replace:

    function (txt) { return txt.charAt(0).toUpperCase() +txt.substr(1); });

with

    txt => txt.charAt(0).toUpperCase() + txt.substr(1)

This is the new ES6 syntax